app/assets/javascripts/sufia/autocomplete/language.es6

```diff
@@ -0,0 +1,25 @@
+export class Language {
```

Comment:
yes, there's room for refactoring here, but I wanted to leave open the possibility of customizing the options per class.